judo-ui-react/src/main/resources/actor/src/config/layout.ts.hbs

```diff
@@ -1,3 +1,56 @@
 {{> fragment.header.hbs }}
 
-export const dividerHeight = 70;
+export const DIVIDER_HEIGHT = 70;
+export const HORIZONTAL_MAX_ITEM = 6;
+export const DRAWER_WIDTH = 260;
+
+export type I18n = 'en' | 'fr' | 'ro' | 'zh'; // 'en' - English, 'fr' - French, 'ro' - Romanian, 'zh' - Chinese
```

Comment:
Why fr / ro / zh is defined here when no resource file added for that languages?

Reply:
Removed bogus examples, added comment to explain the intention.